README.md

```diff
@@ -125,19 +63,29 @@ git clone https://github.com/sonyco-4u/blog-app.git
 
 cd blog-app
 
+### Installation
+
+Install this project with:
+
+```
+- bundle install
+```
+
 ### Usage
 
 To run the project, execute the following command:
 
-rails server
+```
+- railser server -p 4001
```

Comment:
### Optional
- [ ] It would be better if we fix the typo in `rails server -p 4001` instead of `railser server -p 4001`, this command will obviously throw an error. Also, before we run the server, it would be better to run our migrations first `rails  db:create db:migrate`

Reply:
Thanks @addod19 for the corrections